v5analytics/visallo
# Fix ontology repository to return parent properties

Whenever the the vertexium ontology repository would query for a parent concept, it would not return the properties that were associated with that concept. This is important when checking inheritance in the case of the generated ontology code, and would cause the IngestRepository to start throwing failures because it tries to prevent invalid data from going into Visallo. By returning a more fully inflated parent concept the VertexiumOntologyRepository not only matches what is expected, but now works the same as in in memory implementation


Testing Instructions:
(These instructions may not be reasonable to actually test)

Generate code for entities of ontology that has two concepts, one parent and one child, with a property on the parent.
Ensure that you are using the VertexiumOntologyRepository
Try and ingest the entities using the generated code by setting a property on the child-concept that is in on the parent concept. Use the IngestRepository to save the created data. There will be no validation errors.

Points of Regression: 
N/A

CHANGELOG
Fixed: VertexiumOntologyRepository returns properties on the parent concept when returning the parent concept
